packages/aws-cdk/package.json

```diff
@@ -39,6 +40,7 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
+    "@aws-cdk/core": "1.22.0",
```

Comment:
Danger Will Robinson! I don't think we want the CLI to depend on the framework, do we?

What is being used from this, and shouldn't it be in the cx-api instead?

Reply:
Relax Will. This is a devDependency needed only to compile the new test, written in TypeScript, which instantiates `App` from `@aws-cdk/core` for the test CDK app that is deployed using the newly created/updated bootstrap Stack. Here: https://github.com/aws/aws-cdk/pull/5856/files?file-filters%5B%5D=#diff-050637e62993e0e8bf23f8ad10fb636b

Reply:
Got it, thanks.